app/controllers/AccountManagement.scala

```diff
@@ -118,15 +118,22 @@ object AccountManagement extends Controller with LazyLogging {
       pendingHolidays = pendingHolidayRefunds(newHolidays)
```

Comment:
Ok I'll fix that when I rebase https://github.com/guardian/subscriptions-frontend/pull/547 to master and resubmit it.